lib/game.rb

```diff
@@ -46,7 +58,7 @@ def valid_move(index)
   end
 
   def end_game
-    if @board.new_board[8] == 'X'
+    if @board.new_board[8] == 'X' || @board.new_board[8] == 'O'
```

Comment:
There's an issue with the logic here. If I pick square 9 (index 8) as my first move, the game ends immediately:

```
> ruby lib/runner.rb
Welcome to Tic Tac Toe

 1 | 2 | 3
---+---+---
 4 | 5 | 6
---+---+---
 7 | 8 | 9

***Player X*** Select a value between 1 and 9!
9

 1 | 2 | 3
---+---+---
 4 | 5 | 6
---+---+---
 7 | 8 | X

game over
```

This end-of-game logic is _close_, but it should probably be something more about there being no more moves that can be made.

Reply:
Based on conversation, this is good to go. We'll deal with this in https://trello.com/c/7rWM1caP/11-ruby-ttt-user-sees-if-game-won-or-tied